### PR TITLE
add WSL to use secretflow in windows

### PR DESCRIPTION
**Pull request by Mingbo-Lee**
add how to install secretflow on windows by WSL so that a windows user can try or use  secretflow.

### CODE REVIEW

**Comment by Mingbo-Lee** [comment]
I have read the CLA Document and I hereby sign the CLA

File: INSTALLATION.md [hunk 1]
```diff
@@ -73,7 +73,41 @@ python setup.py bdist_wheel
 pip install dist/*.whl
 ```
 
+## Option 4: from WSL
+
+SecretFlow does not support windows directly now,however, a windows user  can use SecretFlow by WSL(Windows Subsystem for Linux)
+
+1. install WSL2 on Windows
```
Comment (zhouaihui) [inline INSTALLATION.md:80]: `install` -> `Install`

File: INSTALLATION.md [hunk 2]
```diff
@@ -73,7 +73,41 @@ python setup.py bdist_wheel
...
+
+- you are supposed to follow the [guide_zh](https://learn.microsoft.com/zh-cn/windows/wsl/install)  or [guide_en](https://learn.microsoft.com/en-us/windows/wsl/install)  to install WSL(Windows Subsystem for Linux)  on your windows and be sure that the version of WSL is
```
Comment (zhouaihui) [inline INSTALLATION.md:82]: `you` -> `You`

Comment (zhouaihui) [inline INSTALLATION.md:82]: `Linux)  on your windows` -> `Linux) on your windows`, extra blank space

File: INSTALLATION.md (hunk 1 above)
Comment (zhouaihui) [reply INSTALLATION.md:80]: Capitalize the first letter, thks. Same below

**Review by zhouaihui — CHANGES_REQUESTED** [review]

**Comment by zhouaihui** [comment]
hi, @Mingbo-Lee , thanks for this improvement.
LGTM, just left some extra  blank space, could you check it again?

**Comment by Mingbo-Lee** [comment]
Hello! @zhouaihui, Thank you very much for your advice. I have checked the spaces word by word and fixed some syntax errors. Would you please review my PR again?

**Review by zhouaihui — APPROVED** [review]
LGTM

**Pull request merged**